### PR TITLE
Fix linting and update dependencies

### PR DESCRIPTION
**Pull request by rtznprmpftl**
Fix the new warnings  from https://github.com/ansible/ansible-lint/issues/1812

And since i touched almost all files i also did a few minor cleanups

### CODE REVIEW

File: roles/buildbot/tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (Akira25) [inline roles/buildbot/tasks/main.yml:1]: It#s fine changing this file, though it will be changed again by #30

**Review by Akira25 — DISMISSED** [review]
Looks good to me.

**Comment by Akira25** [comment]
Please resolve the conflict. Feel free to merge then.

**Comment by rtznprmpftl** [comment]
> Please resolve the conflict. Feel free to merge then.

I can't. Every change needs an approval  ;-)

**Pull request merged**